spec/components/uicontainer.spec.ts

```diff
@@ -0,0 +1,24 @@
+import { MockHelper, TestingPlayerAPI } from '../helper/MockHelper';
+import { UIContainer } from '../../src/ts/components/uicontainer';
+
+let playerMock: TestingPlayerAPI;
+
+describe('UIContainer', () => {
+  beforeEach(() => {
+    playerMock = MockHelper.getPlayerMock();
+  });
+
+  describe('release', () => {
+    it('checks if userInteractionEvents and uiHideTimeout is undefined', () => {
```

Comment:
I think this test needs to be split in two. The first one testing if `release` does not crash, and the second one if `userInteractionEvents` and `uiHideTimeout` are undefined, which only makes sense if  `configure` is called (without `configure` they will always be undefined).

Reply:
Although, `release` only crashed if those properties were undefined, right? Anyway without calling `configure` the test does not cover the actual issue we had. So maybe two tests which check if `release` works with and without `configure`.

Reply:
> Although, release only crashed if those properties were undefined, right?

Yes